databases/week1/db-week1.sql

```diff
@@ -0,0 +1,84 @@
+/*Find out how many tasks are in the task table*/
```

Comment:
```suggestion
--Find out how many tasks are in the task table
```
--You can use single line comment here.